README.md

```diff
@@ -13,6 +13,8 @@ A tool to safeguard the welfare of Western Health anaesthetists during COVID-19
 
 Currently under development by a community of volunteers.
 
+If you've never used Github here's a [Github101.md](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/blob/master/GITHUB101.md) to get you started with the basics.
```

Comment:
There is a way to reference files in the same repo.  Have a look at https://guides.github.com/features/mastering-markdown/

I think its just like :
```suggestion
If you've never used Github here's a [Github101.md](docs/GITHUB101.md) to get you started with the basics.
```